android/app/build.gradle

```diff
@@ -222,6 +222,7 @@ repositories {
 }
 
 dependencies {
+    compile project(':lottie-react-native')
```

Comment:
Can you change this `compile` to `implementation`? Unfortunately react native still generates the older gradle syntax (`compile`), so we need to manually change this.

Reply:
done